irods_consortium_continuous_integration_build_hook.py

```diff
@@ -34,11 +33,22 @@ def install_building_dependencies():
         'irods-externals-json3.7.3-0',
         'irods-externals-libarchive3.3.2-1',
         'irods-externals-zeromq4-14.1.6-0'
-        ])
+        ]
+    if externals_directory is None and externals_directory == 'None':
```

Comment:
I'm confused. Does this ever result in being `true`?

Reply:
thank you for catching it.